lighthouse-core/runner.js

```diff
@@ -36,6 +37,13 @@ class Runner {
       return Promise.reject(err);
     }
 
+    const sentryContext = Sentry.init(opts);
+    Sentry.captureBreadcrumb({
+      message: 'Run started',
+      category: 'lifecycle',
+      data: sentryContext,
+    });
```

Comment:
the point of the breadcrumb was to get easy access to the context (URL, throttling settings, etc), I'm not sure what we'd want to track at the end since we didn't have anything fatal along the way :)